scripts/test-ssr/src/commands/main.ts

```diff
@@ -63,7 +63,11 @@ function interceptConsoleLogs() {
 }
 
 export async function main(params: MainParams) {
-  const distDirectory = path.resolve(process.cwd(), 'node_modules', '.cache', 'ssr-tests');
+  /**
+   * dist directory cannot be under node_modules in order to make TS path aliases work.
+   * @see https://github.com/evanw/esbuild/blob/main/CHANGELOG.md#0180
+   */
+  const distDirectory = path.resolve(process.cwd(), 'dist', 'ssr-tests');
```

Comment:
thoughts  @layershifter  ? 

we can also put it within `<package-root>/.cache/ssr-test/*`

but as we already have `/dist` I think not introducing another directory churn will be for the best. WDYT ?

Reply:
I don't mind, we also publish only `"dist/*.d.ts",`, so it should be fine 👍

Reply:
sounds good, ty!